src/Umbraco.Infrastructure/Packaging/PackageDataInstallation.cs

```diff
@@ -57,29 +52,25 @@ public class PackageDataInstallation
             PropertyEditorCollection propertyEditors,
             IScopeProvider scopeProvider,
             IShortStringHelper shortStringHelper,
-            IOptions<GlobalSettings> globalSettings,
             IConfigurationEditorJsonSerializer serializer,
             IMediaService mediaService,
-            IMediaTypeService mediaTypeService,
-            IHostingEnvironment hostingEnvironment)
```

Comment:
I think we should add a constructor overload and obsolete the old one here, and schedule the removal for a later release, since it's a breaking change that can easily be avoided,

Reply:
Agreed, we should keep the breaking changes to a minimum, but this unfortunately wasn't that easy 😞 Had to register `PackageDataInstallation` using a factory method and add `[ActivatorUtilitiesConstructor]` to `LanguageController` for everything to work again... Fixed now though!